packages/crawler/reports/2022-12-19_data-aggregated.json

```diff
@@ -0,0 +1,1400 @@
+                            "logo-facebook": 2,
+                            "logo-instagram": 2,
+                            "logo-pinterest": 2,
+                            "logo-twitter": 2,
+                            "logo-linkedin": 2,
+                            "external": 6,
+                            "": 7,
+                            "logo-youtube": 1,
+                            "logo-xing": 1
+                        }
+                    },
+                    "color": {
+                        "amount": 136,
+                        "values": {
+                            "inherit": 126,
+                            "default": 10
+                        }
+                    },
+                    "size": {
+                        "amount": 136,
+                        "values": {
+                            "inherit": 84,
+                            "small": 51,
+                            "medium": 1
+                        }
+                    },
+                    "lazy": {
+                        "amount": 136,
+                        "values": {
+                            "false": 136
+                        }
+                    },
+                    "theme": {
+                        "amount": 136,
+                        "values": {
+                            "light": 135,
+                            "dark": 1
+                        }
+                    },
+                    "source": {
+                        "amount": 3,
+                        "values": {
+                            "/assets/icons/search-agent.svg": 1,
+                            "/_next/static/media/multicolor.0794a88f.svg": 1,
+                            "https://nav.porsche.com/00BC524/footer/assets/flags/CH.svg": 1
+                        }
+                    }
+                },
+                "unusedProperties": [
+                    "isDelegatingFocus",
+                    "isInternal",
+                    "isThemeable",
+                    "props",
+                    "hasSlot",
+                    "hasSlottedCss",
+                    "hasEvent",
+                    "hasAriaProp",
+                    "hasObserveAttributes",
+                    "hasObserveChildren",
+                    "styling"
+                ]
+            },
+            "p-flex": {
+                "amount": 30,
+                "hostPdsComponent": 0,
+                "slot": 30,
+                "properties": {
+                    "inline": {
+                        "amount": 30,
+                        "values": {
+                            "false": 29,
+                            "true": 1
+                        }
+                    },
+                    "wrap": {
+                        "amount": 30,
+                        "values": {
+                            "nowrap": 25,
+                            "wrap": 4,
+                            "{base: 'wrap', xs: 'nowrap'}": 1
```

Comment:
it's not a valid json - we should use "JSON.stringify" in order to have a unique format

Reply:
Investigated is: it's the way it's implemented on https://www.porsche.com/swiss/de/. They put `{base: 'wrap', xs: 'nowrap'}` as a value of "size" attribute. The questions are:

- is it working like this (is the size still being applied, even if it's not a correct json)?
- if it's working - should I then convert this value to an object? (by making it to be correct json format)

TODO: discuss with the team

Reply:
Discussed with the team: we should parse this object using existing functionality (parseJSON)

Reply:
Done